VerilogCodingStyle.md

```diff
@@ -2543,8 +2569,11 @@ initialization. For example:
 &#x1f44d;
 ```systemverilog {.good}
 wire [7:0] sum = a + b;  // Continuous assignment
+```
 
-logic [7:0] acc = '0;  // Initialization
+&#x1f44e;
```

Comment:
👍  yes thank you this should not be suggested, might have been a leftover when this contained the DV-friendly portions

Reply:
I agree that initializations shouldn't appear as "good code example" as shown currently, so removing it is a good thing.

However, in the FPGA world, there are valid use cases for initializations (since LUTs are actually initialized this can reduce area by saving reset logic). Without going into that discussion, I'd simply drop the example and not explicitly add it as "bad" example either.

Reply:
How about this?
This takes care of both cases.